slick.grid.js

```diff
@@ -2726,6 +2728,23 @@ if (typeof Slick === "undefined") {
       return columns;
     }
 
+    function getColumnById(columnId, useOnlyVisibleColumns){
+      var columnList = useOnlyVisibleColumns?getColumns():getAllColumns(true);
```

Comment:
can you space this code, it would be a bit more readable

Reply:
sure